taskcluster/ci/beetmover/kind.yml

```diff
@@ -0,0 +1,35 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: mozillavpn_taskgraph.loader.multi_dep:loader
+
+transforms:
+    - mozillavpn_taskgraph.transforms.multi_dep:transforms
+    - mozillavpn_taskgraph.transforms.beetmover:transforms
+    - taskgraph.transforms.task:transforms
+
+group-by: build-type
+
+only-for-build-types:
+    - macos/opt
+
+kind-dependencies:
+    - signing
```

Comment:
This will get a little bit weird because on Windows the kind we will be beetmoving is `repackage-signing`.. but I guess we can deal with that later.

Reply:
Ah yeah, I dunno, maybe add another kind that depends on repackage-signing and put it thru the same beetmover transform? Seems kinda hacky

Reply:
We might want to make it not use the `multi_dep` loader and just use the transform loader directly with each task explicitly listing a dependency.. but we can punt that to when we stand up the Windows task.

Reply:
That sounds better